Makefile

```diff
@@ -18,7 +18,7 @@ env:
 
 prepare: env
 
-	go get -u -v github.com/spf13/hugo
+	go get -u -v github.com/gohugoio/hugo
```

Comment:
with Go 1.16 we could probably do `go install github.com/gohugo10/hugo@latest` to use go modules

Reply:
You're right! It's better and faster.